manifest_staging/charts/gatekeeper/README.md

```diff
@@ -223,9 +223,14 @@ information._
 | audit.channel                                              | (alpha) Channel name for exporting audit violation messages                                                                                                                                                                                                                                   | `audit-channel`                                                                                                                                                       |
+| audit.exportVolume                                         | (alpha) Volume for audit pod to export violations.                                                                                                                                                                                                                                   | `{"name":"tmp-violations","emptyDir":{}}`                                                                                                                                                       |
+| audit.exportVolumeMount.path                                         | (alpha) VolumeMount for audit pod manager container to export violations and sidecar container to read from.                                                                                                                                                                                                                                   | `/tmp/violations`                                                                                                                                                       |
+| audit.exportSidecar                                        | (alpha) Sidecar container to read violations from disk.                                                                                                                                                                                                                                   | `{"name":"reader","image":"ghcr.io/open-policy-agent/fake-reader:latest","imagePullPolicy":"Always","securityContext":{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"readOnlyRootFilesystem":true,"runAsGroup":999,"runAsNonRoot":true,"runAsUser":1000,"seccompProfile":{"type":"RuntimeDefault"}},"volumeMounts":[{"mountPath":"/tmp/violations","name":"tmp-violations"}]}`                                                                                                                                                       |
```

Comment:
where is the code that pushes `ghcr.io/open-policy-agent/fake-reader`

Reply:
For now I have pushed them manually before building a CI for that.

Reply:
As a follow up, please add this to CI. I can see maybe it could have CVEs in the future that we may need to patch.

Reply:
+1 This should be added for all sample images as follow up.